src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -462,7 +462,8 @@ Status GcsActorManager::CreateActor(const ray::rpc::CreateActorRequest &request,
 
   auto iter = registered_actors_.find(actor_id);
   if (iter == registered_actors_.end()) {
-    RAY_LOG(INFO) << "Actor " << actor_id << " may be already destroyed.";
+    RAY_LOG(INFO) << "Actor " << actor_id
```

Comment:
This probably should be DEBUG? I don't think non-developers can get any information from this log.